roles/sssd_ad/tasks/main.yml

```diff
@@ -0,0 +1,124 @@
+---
+- name: Sssd_ad | change hostname to match AD
```

Comment:
Can we add a comment about the reason for this (that Windows limits hostnames to 12 characters).